scripts/examples/gn_efr32_example.sh

```diff
@@ -191,14 +194,17 @@ else
             #   optArgs+="openthread_root=\"//third_party/connectedhomeip/third_party/openthread/ot-efr32/openthread\" openthread_efr32_root=\"//third_party/connectedhomeip/third_party/openthread/ot-efr32/src/src\""
             #    shift
             #    ;;
-            --no-version)
-                USE_GIT_SHA_FOR_VERSION=false
-                shift
-                ;;
             --release)
                 optArgs+="is_debug=false disable_lcd=true chip_build_libshell=false enable_openthread_cli=false use_external_flash=false chip_logging=false silabs_log_enabled=false "
                 shift
                 ;;
+
+            *"sl_matter_version_str="*)
+                optArgs+="$1"
```

Comment:
are you sure this works as intended and still works with other arguments after it?  You at least need to add a space after it to deal with the following arguments I believe.

Reply:
Where does it need a space? I tested this with an argument after and seems to work as intended

Reply:
we keep doing`optArgs+=` parsing through arguments, if the is no space after the $1, the arguments will be clump up together and considered as one arg

for example if I do in my command `sl_matter_version_str="1.1.0" chip_build_libshell=false `
`optArgs` will =  `sl_matter_version_str="1.1.0"chip_build_libshell=false` without a space between them 

all the other optArgs+= have a space added after it $1. am I missing something?

Reply:
Added the space. optArgs indeed had clumped arguments, but gn still did the intended behavior.

Reply:
gn might have passed for your set of arguments but it would not have resulted in the right outcome. I tested it locally the version string became formed with all the following arguments passed. for other arguments that do not expect a string it just errors out directly